src/modules/vtol_att_control/standard_params.c

```diff
@@ -87,10 +87,25 @@ PARAM_DEFINE_FLOAT(VT_FWD_THRUST_SC, 0.7f);
 PARAM_DEFINE_FLOAT(VT_B_TRANS_RAMP, 3.0f);
 
 /**
- * Output on airbrakes channel during back transition
+ * Option for use of Flaperons on back transition.
+ *
+ * Used for enabling Flaperons during a back transition.
+ * Flaps and Airbrakes need to be enabled for your selected model/mixer
+ *
+ * @min 0
+ * @max 1
+ * @increment 0.01
+ * @decimal 2
+ * @group VTOL Attitude Control
+ */
+PARAM_DEFINE_FLOAT(VT_B_FL_USE, 1.0f);
```

Comment:
Wonder if we even need this param, or always set the flaperons (or should we start calling them "spoilerons") and then leave it to the mixer to enable them on a particular vehicle.

Reply:
I added it simply because while you can do it in the mixer, there is one for fixed wing side so giving you the choice that is front facing is easier than editing the mixer or control allocation matrix but im ok with removing it, just figured same as before, if granular functionality is there on FW side, might as well match it on VTOL side.

Reply:
Right, prob we need to keep it.